compiler/msgs.nim

```diff
@@ -269,6 +269,7 @@ const
     errExprHasNoAddress: "expression has no address",
     errXStackEscape: "address of '$1' may not escape its stack frame",
     errVarForOutParamNeededX: "for a \'var\' type a variable needs to be passed; but '$1' is immutable",
+    errExprIsNoException: "raised object does not inherit from Exception",
```

Comment:
We don't do this anymore. Use the version of ``localError`` that takes a string.